perception/lidar_centerpoint/lib/postprocess/postprocess_kernel.cu

```diff
@@ -107,6 +107,31 @@ __global__ void generateBoxes3D_kernel(
   det_boxes3d[idx].yaw = atan2f(yaw_sin, yaw_cos);
   det_boxes3d[idx].vel_x = vel_x;
   det_boxes3d[idx].vel_y = vel_y;
+
+  if (has_variance) {
+    const float offset_x_variance = out_offset[down_grid_size * 2 + idx];
+    const float offset_y_variance = out_offset[down_grid_size * 3 + idx];
+    const float z_variance = out_z[down_grid_size * 1 + idx];
+    const float w_variance = out_dim[down_grid_size * 3 + idx];
+    const float l_variance = out_dim[down_grid_size * 4 + idx];
+    const float h_variance = out_dim[down_grid_size * 5 + idx];
+    const float yaw_sin_log_variance = out_rot[down_grid_size * 2 + idx];
+    const float yaw_cos_log_variance = out_rot[down_grid_size * 3 + idx];
+    const float vel_x_variance = out_vel[down_grid_size * 2 + idx];
+    const float vel_y_variance = out_vel[down_grid_size * 3 + idx];
+
+    det_boxes3d[idx].x_variance = voxel_size_x * downsample_factor * expf(offset_x_variance);
+    det_boxes3d[idx].y_variance = voxel_size_x * downsample_factor * expf(offset_y_variance);
+    det_boxes3d[idx].z_variance = expf(z_variance);
+    det_boxes3d[idx].length_variance = expf(l_variance);
+    det_boxes3d[idx].width_variance = expf(w_variance);
+    det_boxes3d[idx].height_variance = expf(h_variance);
+    det_boxes3d[idx].yaw_variance = (powf(yaw_cos, 2) * expf(yaw_sin_log_variance) +
+                                     powf(yaw_sin, 2) * expf(yaw_cos_log_variance)) /
+                                    (powf((powf(yaw_sin, 2) + powf(yaw_cos, 2)), 2));
```

Comment:
Avoid using powf
```suggestion
    const float yaw_sin_sq = yaw_sin * yaw_sin;
    const float yaw_cos_sq = yaw_cos * yaw_cos;
    const float yaw_norm_sq = yaw_sin_sq + yaw_cos_sq;
    det_boxes3d[idx].yaw_variance = (yaw_cos_sq * expf(yaw_sin_log_variance) +
                                     yaw_sin_sq * expf(yaw_cos_log_variance)) / yaw_norm_sq;
```

Reply:
Thank you for the comment.  It is fixed in https://github.com/autowarefoundation/autoware.universe/pull/6573/commits/8a2373163811186045c81cf3c71f7ed28831e92c.